src/vset.h

```diff
@@ -0,0 +1,317 @@
+#ifndef VOLATILESET_H
+#define VOLATILESET_H
+
+#include <stddef.h>
+#include <stdbool.h>
+
+#include "hashtable.h"
+#include "rax.h"
+#include "sds.h"
+#include "monotonic.h" /* for mstime_t*/
```

Comment:
Comment is incorrect.  Monotonic isn't needed in this .h file.

Reply:
Correct. A leftover. Not sure if we fixed that on the later pr. Of not will fix it.